src/components/formio/columns/columns.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import {Columns, IColumnsComponent, IContentComponent} from '@components';
+import {IRenderComponentProps, RenderComponent} from '@lib/renderer';
+import {expect} from '@storybook/jest';
+import type {ComponentStory, Meta} from '@storybook/react';
+import {within} from '@storybook/testing-library';
+import {Formik} from 'formik';
+import React from 'react';
+
+const meta: Meta<typeof Columns> = {
+  title: 'Components / Formio / Columns',
+  component: Columns,
+  decorators: [],
+  parameters: {},
+};
+export default meta;
+
+export const columns: ComponentStory<React.FC<IRenderComponentProps<IColumnsComponent>>> = args => (
+  <RenderComponent {...args} />
+);
+columns.args = {
+  component: {
+    key: undefined,
+    type: 'columns',
+    columns: [
+      {
+        size: 9,
+        components: [{type: 'content', html: 'Left column.'} as IContentComponent],
```

Comment:
Looking at https://open-formulieren.github.io/open-forms-sdk/?path=/story/form-io-components-content--default-story&globals=locale:nl;locales.nl:Nederlands;locales.en:English and knowing how the form builder works (it always asks for a `key` property), I'd add `label` and `key` here too, otherwise you risk people copy pasting incomplete content component definitions.

Is the cast here really necessary btw, shouldn't this be possible with just the object definition without any casts?